views/update/update-beneficial-owner-type.html

```diff
@@ -94,8 +94,7 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
     <form method="post">
       {% include "includes/csrf_token.html" %}
       {% include "includes/beneficial-owner-type-add-radios.html" %}
-
-      {% if (feature_flag_relevant_period === "true") and (update.change_bo_relevant_period === "CHANGE_BO_RELEVANT_PERIOD" or update.trustee_involved_relevant_period === "TRUSTEE_INVOLVED_RELEVANT_PERIOD" or update.change_beneficiary_relevant_period === "CHANGE_BENEFICIARY_RELEVANT_PERIOD") %}
+      {% if (feature_flag_relevant_period === "true") and (update.change_bo_relevant_period === "CHANGE_BO_RELEVANT_PERIOD") %}
```

Comment:
If we include this change, you can only select the first checkbox for this case. We want any of the checkboxes to trigger relevant period content